pkg/hive/manager.go

```diff
@@ -43,6 +44,7 @@ type ClusterManager interface {
 	IsClusterInstallationComplete(ctx context.Context, doc *api.OpenShiftClusterDocument) (bool, error)
 	GetClusterDeployment(ctx context.Context, doc *api.OpenShiftClusterDocument) (*hivev1.ClusterDeployment, error)
 	ResetCorrelationData(ctx context.Context, doc *api.OpenShiftClusterDocument) error
+	GetSyncSetResources(ctx context.Context, doc *api.OpenShiftClusterDocument) (*hivev1alpha1.ClusterSync, error)
```

Comment:
Just a nit about naming: If GetSyncSetResources only deals with ClusterSync objects, can we name it accordingly ? Like GetClusterSync ?

Reply:
Function renamed to `GetClusterSync`